projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,3379 @@
+
+	"quark.config.mobs.toretoise.name": "矿石陆龟",
+	"quark.config.mobs.toretoise.desc": "矿石陆龟是一种新的中立生物，会在完全黑暗的地下深处生成。生成时，它的背甲会随机覆盖一种矿石（煤、铜、铁、青金石或红石）。你可以用镐攻击矿石陆龟，它会掉落背上的矿石！\n你攻击矿石陆龟时，如果用的不是镐，或者它没有矿石，它会发动震地攻击。这攻击会伤害周围的任何非矿石陆龟的实体，还会将它们的仇恨转移到你身上。\n只要小心一些，就可以将矿石陆龟带回家去喂养，以便它生长出更多的矿石。\n(1) 喂食矿石陆龟发光浆果，有时能让它的矿石再生。\n(2) 矿石陆龟每次再生矿石时，都会随机挑选一种。\n(3) 如果启用了饲料槽（来自自动化类别），矿石陆龟会从中取食。\n(4) 矿石陆龟无法繁殖。但它们是像敌对生物一样生成的，因此你仍然可以得到更多矿石陆龟。\n(5) 当矿石陆龟生成矿石时，它会进入1分钟的进食冷却时间。\n(6) 矿石陆龟会沉入水中，它们免疫溺水、摔落和火焰伤害。\n(7) 矿石陆龟一般1格高，背上有矿石时，会长到1.4格高。\n(8) 矿石陆龟移动非常缓慢，又不能用拴绳牵引，建议使用矿车运输。\n(9) 如果启用了铁杆（来自自动化类别），用活塞将它们推向矿石陆龟，可以破坏其背上的矿石。\n注意：1.18之前，矿石陆龟是用洞穴根茎（来自世界类别）喂食的（如果被禁用，则用仙人掌喂食）。它们的生成也不需要完全黑暗。",
+	"quark.config.mobs.toretoise.allow_toretoise_to_regrow.name": "允许矿石陆龟再生矿石",
+	"quark.config.mobs.toretoise.cooldown_ticks.name": "冷却时间",
+	"quark.config.mobs.toretoise.cooldown_ticks.desc": "挖走矿石陆龟背上的矿物后经过多久，喂养它才能使其矿物再生。单位为刻。",
+	"quark.config.mobs.toretoise.foods.name": "食物列表",
+	"quark.config.mobs.toretoise.foods.desc": "可以用来喂养矿石陆龟，使其矿物再生的物品。",
+	"quark.config.mobs.toretoise.max_y_level.name": "最大可生成高度",
+	"quark.config.mobs.toretoise.regrow_chance.name": "矿石再生几率",
+	"quark.config.mobs.toretoise.regrow_chance.desc": "矿石陆龟冷却后，喂食它有几率让它们再生矿石，几率为该值的倒数。\n设为1必定再生矿石，该值越高=几率越低。",
+	"quark.config.mobs.toretoise.dimensions.name": "维度配置",
+	"quark.config.mobs.toretoise.dimensions.dimensions.name": "维度列表",
+	"quark.config.mobs.toretoise.dimensions.is_blacklist.name": "用作黑名单",
+	"quark.config.mobs.toretoise.spawn_config.name": "生成配置",
+	"quark.config.mobs.toretoise.spawn_config.max_group_size.name": "群体规模上限",
+	"quark.config.mobs.toretoise.spawn_config.max_group_size.desc": "允许的值：[1,)",
+	"quark.config.mobs.toretoise.spawn_config.min_group_size.name": "群体规模下限",
+	"quark.config.mobs.toretoise.spawn_config.min_group_size.desc": "允许的值：[1,)",
+	"quark.config.mobs.toretoise.spawn_config.spawn_weight.name": "生成权重",
+	"quark.config.mobs.toretoise.spawn_config.spawn_weight.desc": "允许的值：(0,)",
+	"quark.config.mobs.toretoise.spawn_config.biomes.name": "生物群系配置",
+	"quark.config.mobs.toretoise.spawn_config.biomes.biomes.name": "生物群系",
+	"quark.config.mobs.toretoise.spawn_config.biomes.biomes.desc": "应该在哪些生物群系中生成矿石陆龟。\n必须要同时符合此项和「生物群系标签」才能生成。",
+	"quark.config.mobs.toretoise.spawn_config.biomes.biomes.biomes.name": "生物群系列表",
+	"quark.config.mobs.toretoise.spawn_config.biomes.biomes.is_blacklist.name": "用作黑名单",
+	"quark.config.mobs.toretoise.spawn_config.biomes.tags.name": "生物群系标签",
+	"quark.config.mobs.toretoise.spawn_config.biomes.tags.desc": "应该在带有哪些标签的生物群系中生成矿石陆龟。\n必须要同时符合此项和「生物群系」才能生成。",
+	"quark.config.mobs.toretoise.spawn_config.biomes.tags.biome_tags.name": "生物群系标签列表",
+	"quark.config.mobs.toretoise.spawn_config.biomes.tags.is_blacklist.name": "用作黑名单",
+
+	"quark.config.mobs.wraith.name": "游魂",
+	"quark.config.mobs.wraith.desc": "游魂是一种新的中立生物，会在灵魂沙谷生成。它们是源自不同生物的迷失灵魂，因此，它们的声音是随机普通生物声音的降调版本。\n它们像普通僵尸一样攻击，给予缓慢效果，并在造成伤害时向后跳回去。死亡后，它们会掉落一颗灵魂珠。\n使用灵魂珠，会释放出困在其中的灵魂，它会飞向最近的下界要塞。",
```

```suggestion
	"quark.config.mobs.wraith.desc": "游魂是一种新的敌对生物，会在灵魂沙谷生成。它们是源自不同生物的迷失灵魂，因此，它们的声音是随机普通生物声音的降调版本。\n它们像普通僵尸一样攻击，给予缓慢效果，并在造成伤害时向后跳回去。死亡后，它们会掉落一颗灵魂珠。\n使用灵魂珠，会释放出困在其中的灵魂，它会飞向最近的下界要塞。",
```
应该不是中立，我都被打死好几次了。
源代码能找到（但不贴了，这几天网络似乎与我有仇）

---

6，wiki上写的是neutral就离谱
不过mc百科上倒是攻击性生物